ci/chains/gaia/v5.0.0/ibc-0/config/genesis.json

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
Should the path to these files be named `.../v5.0.5/` ? Instead of `v5.0.0`.

Reply:
Ignore my comment above. I just realized now we originally wanted in this PR to add to CI gaiad v5.0.0 (hence the config files for `v5.0.0`) and eventually added it for `v5.0.5`. I think we should delete all files related to `v5.0.0` since we're not using those.

Reply:
I just realized now we originally wanted in this PR to add to CI gaiad v5.0.0 (hence the config files for v5.0.0) and eventually added it for v5.0.5.

I think we should delete all files related to v5.0.0 since we're not using those.

Reply:
@adizere removed the v5.0.0 files and updates PR description to say it updates to v5.0.5